test/addons/dlopen-ping-pong/binding.gyp

```diff
@@ -0,0 +1,15 @@
+{
+  'targets': [
+    {
+      'target_name': 'ping',
+      'type': 'shared_library',
+      'defines': [ 'V8_DEPRECATION_WARNINGS=1' ],
```

Comment:
Forgot to mention, this define is not strictly necessary.  It doesn't hurt either, though.